README.md

```diff
@@ -67,14 +67,18 @@ exportedTagsOnMetrics:
 
 ### Auto-discovery job
 
-| Key                  | Description                                                                              |
-| -------------------- | ---------------------------------------------------------------------------------------- |
-| region               | AWS region                                                                               |
-| type                 | Service name, e.g. "ec2", "s3", etc.                                                     |
-| roleArn              | IAM role to assume (optional)                                                            |
-| searchTags           | List of Key/Value pairs to use for tag filtering (all must match), Value can be a regex. |
-| metrics              | List of metric definitions                                                               |
-| additionalDimensions | List of dimensions to return beyond the default list per service                         |
+| Key                  | Description                                                                                         |
+| -------------------- | --------------------------------------------------------------------------------------------------  |
+| region               | AWS region                                                                                          |
+| type                 | Service name, e.g. "ec2", "s3", etc.                                                                |
+| enableMetricData     | Use Cloudwatch GetMetricData Api insisted of getmetricstatistics.                                   |
```

Comment:
I think this line should probably read "Use CloudWatch GetMetricData API instead of GetMetricStatistics"

Reply:
fixed